content/rdi/monitoring-guide.md

```diff
@@ -130,6 +95,6 @@ This list shows exported RDI metrics along with their descriptions:
 
 | Metric Name                 | Labels                                                                                                     | Values               | Description                                                                                                                                                                                                        |
 | --------------------------- | ---------------------------------------------------------------------------------------------------------- | -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| rdi_engine_status           | {status=RUNNING \| STOPPED}                                                                                | 0, 1                 | Status of RDI Engine. 0 - RDI Engine is stopped, 1 - RDI Engine is running.                                                                                                                                        |
+| rdi_engine_state            | {status=RUNNING \| STOPPED, sync_mode=SNAPSHOT \| STREAMING \| UNKNOWN}                                    | 0, 1                 | Status of RDI Engine. 0 - RDI Engine is stopped, 1 - RDI Engine is running.                                                                                                                                        |
```

Comment:
Add description for the `sync_mode` as well.

Reply:
Added